package iegad.net;


message echo_msg {
//TCPÏûÏ¢
required string requ_str = 1;
required string resp_str = 2;
};